getting-started/kubernetes/trying-ebpf.md

```diff
@@ -0,0 +1,228 @@
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- **calico/node**
+- **eBPF dataplane**
+
+### Concepts
+
+#### eBPF
+
+eBPF is a technology that allows you to write mini programs that can be attached to various low-level hooks in the Linux kernel, for a wide variety of uses including networking, security, and tracing. You’ll see a lot of non-networking projects leveraging eBPF, but for {{site.prodname}} our focus is on networking, and in particular, pushing the networking capabilities of the latest Linux kernel’s to the limit.
+
+#### eBPF mode manifest
+
+To make it easier to try out the tech preview, we have created a custom Kubernetes manifest (.yaml file) with eBPF mode pre-enabled.
+
+### Before you begin...
+
+In the tech preview release, eBPF mode has the following pre-requisites:
+
+- A v5.3+ Linux kernel, for the tech preview release, we have limited our testing to Ubuntu 19.10 so we strongly recommend that you start with that too.
+- An underlying network that doesn't require Calico to use an overlay.  The instructions below guide you through setting up a cluster in a single AWS subnet; an alternative would be to set up your cluster on-prem with a routed network topology.
+- The network must be configured to allow VXLAN packets between  {{site.prodname}}  hosts.
+- Single-homed hosts; eBPF mode currently assumes a single "main" host IP and interface.
+- Must use the Calico CNI plugin and Calico IPAM.  It is not yet compatible with third-party CNI plugins (AWS CNI/Azure CNI/GKE CNI/flannel etc).
+- IPv4 only.  The tech preview release does not support IPv6.
+- Kubernetes API Datastore only.
+- Typha is not supported in the tech preview. 
+- The base [requirements]({{site.baseurl}}/getting-started/kubernetes/requirements) also apply.
+
+### How to
+
+- [Set up a suitable cluster](#set-up-a-suitable-cluster)
+- [Install Calico on nodes](#install-calico-on-nodes)
+- [Toggle between eBPF and the standard linux networking pipeline](#toggle-between-ebpf-and-the-standard-linux-networking-pipeline)
+
+#### Set up a suitable cluster
+
+We recommend using `kubeadm` to bootstrap a suitable cluster on AWS.  
+
+1. In AWS, create a controller node and at least 2 worker nodes in the same VPC subnet.  Use Ubuntu 19.10 as the image.
+
+1. **Important** disable the source/destination check on each node's interface.  This is required in order to use non-overlay networking.
+
+   1. Open the Amazon EC2 console and navigate to "Instances".
+
+   1. Select the instance and choose **Actions** > **Networking** > **Change Source/Dest. Check**.
+
+   1. Verify that source/destination check is disabled. Otherwise, choose **Yes, Disable**.
+
+1. On each node, {% include open-new-window.html text='install kubeadm' url='https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/' %}; we recommend using docker as the containeriser in order to match our test environment as closely as possible.
```

Comment:
"containeriser" should probably be "container runtime" - that's how this class of thing is usually described.